tools/cli/helpers/doc-parser/runner.php

```diff
@@ -74,6 +74,51 @@ function get_html_from_markdown( $file_path ) {
 	);
 
 	$doc_title = $file_path;
+	$anchors   = $document->getElementsByTagName( 'a' );
+	foreach ( $anchors as $anchor ) {
+		$link = parse_url( $anchor->getAttribute( 'href' ) );
+		if ( ! $link || isset( $link['host'] ) || ! isset( $link['path'] ) ) {
+			continue;
+		}
+
+		// Replace any relative links with absolute links to the GitHub repo. If it's deeper than 2 levels, it's a link to a file in the repo.
+		if ( str_starts_with( $link['path'], '../' ) ||
+			str_starts_with( $link['path'], '/projects/' ) ||
+			( substr_count( $link['path'], '/' ) > 2 ) ) {
+				$link['path'] = preg_replace( '~^(\./|../)~', '', $link['path'], 1 ); // Remove leading ./ or ../
+				$link['path'] = 'https://github.com/Automattic/jetpack/blob/trunk' .
+					( ! str_starts_with( $link['path'], '/' ) ? '/' : '' ) .
+					$link['path'];
+		}
+
+		// Handle docs that just live in github, ending in anything other than .md.
+		$extension = pathinfo( $link['path'], PATHINFO_EXTENSION );
+		if ( ( $extension !== 'md' || $extension === '' ) &&
+			! str_starts_with( $link['path'], 'http' ) ) {
+				$link['path'] = preg_replace( '~^(\./|/)~', '', $link['path'], 1 ); // Remove leading ./ or /
+				$link['path'] = 'https://github.com/Automattic/jetpack/blob/trunk/' .
+					( str_contains( $link['path'], 'examples/' ) ? 'docs/' : '' ) .
+					$link['path'];
+		}
+
+		// If the Path starts with ./docs/ and contains 2 slashes, it's a relative link to another doc.
+		if ( ( str_starts_with( $link['path'], './docs' ) ||
+			str_starts_with( $link['path'], '/docs/' ) ||
+			str_starts_with( $file_path, 'docs/' ) ) &&
+			substr_count( $link['path'], '/' ) <= 2 ) {
+				$link['path'] = str_replace( array( './docs/', '/docs/', './' ), '', $link['path'] );
+				$link['path'] = 'docs-' . $link['path'];
```

Comment:
This has to start with a slash, otherwise links get stacked on top of the current path.
```suggestion
				$link['path'] = '/docs-' . $link['path'];
```

Reply:
Oh weird, I tested with both methods and the links work fine. I think something else must be handling concatenating it to the path in the href, but I don't think it hurts to be sure.